docs/reference/advanced_configuration.rst

```diff
@@ -450,4 +450,4 @@ by overriding ``checkAccess`` function::
         }
     }
 
-.. _`Core's documentation`: http://sonata-project.org/bundles/core/master/doc/reference/form_types.html#sonata-type-translatable-choice
+.. _`Core's documentation`: https://sonata-project.org/bundles/core/master/doc/reference/form_types.html#sonata-type-translatable-choice
```

Comment:
At `docs/reference/action_list.rst:582` you're changing the target from "SonataCoreBundle" to "form-extensions":
```diff
- .. _`here`: https://github.com/sonata-project/SonataCoreBundle/tree/3.x/src/Form/Type
+ .. _`here`: https://github.com/sonata-project/form-extensions/tree/1.x/src/Type
```
Could we do the same here?

Reply:
The replacement I made was one from one GitHub link to another. The remaining Core bundle link refers to its published documentation, which the form extensions do not have. I think they need to be set up. When that is done, they can be linked. The homepage link in composer.json is https://sonata-project.org/bundles/form-extensions, which does not exist.

Do you have any idea who might be able to do this? Are the sources of https://sonata-project.org/ available anywhere?